test/test_other.py

```diff
@@ -8815,7 +8815,7 @@ def test_exceptions_stack_trace_and_message(self):
       'at (src.wasm.)?foo',
       'at (src.wasm.)?main']
 
-    if '-fwasm-excpeptions' in self.emcc_args:
+    if '-fwasm-exceptions' in self.emcc_args:
```

Comment:
The reason is passed in CI is that we use the emsdk version of node there (v16).

Since version 16 of node doesn't support wasm exceptions the `require_wasm_eh` helper will already force v8 for all wasm exceptions tests when running on CI.

Its only users of node v17 and above that would be effected by this typo.